packages/react-native-codegen/android/gradlePlugin-build/gradlePlugin/src/main/java/com/facebook/react/codegen/plugin/CodegenPlugin.java

```diff
@@ -62,12 +64,13 @@ public void apply(final Project project) {
 
               ImmutableList<String> execCommands =
                   new ImmutableList.Builder<String>()
-                      .add("yarn")
+                      .add(os.contains("windows") ? "yarn.cmd" : "yarn")
```

Comment:
Curious why these are needed as I've always just run `yarn` on Windows. Also seems to match the docs: https://classic.yarnpkg.com/en/docs/install/#windows-stable

Reply:
I honestly don't know - it just wouldn't work otherwise. I was able to google my error and resolve it this way. 
Also, in D:\react1\tb\rn\ReactAndroid\build.gradle@243 there's, which informed my resolution:
```
if (Os.isFamily(Os.FAMILY_WINDOWS)) {
    return "ndk-build.cmd"
} else {
    return "ndk-build"
}
```